render-utils.js

```diff
@@ -0,0 +1,29 @@
+// export funciton renderPoll(poll) {
+//     const pollDiv = document.createElement('div');
+//     const question = document.createElement('p');
+//     question.textContent = 
+
+// }
+
+export function renderPastPoll(pastPoll) {
+    const container = document.createElement('div');
+    const pQuestionEl = document.createElement('p');
+    const pTitleA = document.createElement('p');
+    const pTitleB = document.createElement('p');
+    const pVotesA = document.createElement('p');
+    const pVotesB = document.createElement('p');
+
+  
+    container.classList.add('past-poll');
+    pQuestionEl.textContent = pastPoll.question;
+    pTitleA.textContent = pastPoll.optionA;
+    pTitleB.textContent = pastPoll.optionB;
+    pVotesA.textContent = pastPoll.VotesA;
+    pVotesB.textContent = pastPoll.VotesB;
```

Comment:
Look at the object keys you are accessing on lines 21 and 22.  `VotesA` and `VotesB` do not match the object that you are creating with `makePoll()`, which you then use to push into your pastPolls array.  (Look at capitalization and pluralization)